public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ResourcePicker/NestedEntry.tsx

```diff
@@ -16,6 +16,7 @@ interface NestedEntryProps {
   isSelectable: boolean;
   isOpen: boolean;
   isDisabled: boolean;
+  isSelectionSummary?: boolean;
```

Comment:
I prefer naming this `scrollIntoView` as you did before so it's clear its function